lib/babe/verify_integration_test.go

```diff
@@ -179,7 +181,7 @@ func TestVerificationManager_VerifyBlock_Secondary(t *testing.T) {
 		Body:   nil,
 	}
 	err = vm.VerifyBlock(&block.Header)
-	require.EqualError(t, err, "failed to verify pre-runtime digest: block producer is not in authority set")
+	require.EqualError(t, err, "invalid signature length")
```

Comment:
Why change the expected error?

Reply:
see https://github.com/ChainSafe/gossamer/pull/3669#discussion_r1455861311